tests/utils/aux_utils_tests.py

```diff
@@ -95,6 +95,29 @@ def test_get_im_name():
     nose.tools.assert_equal(im_name, 'im_c2_z3_t1_p4_hej.png')
 
 
+def test_get_sms_im_name():
+    im_name = aux_utils.get_sms_im_name(
+        time_idx=0,
+        channel_name='phase',
+        slice_idx=10,
+        pos_idx=100,
+        extra_field='blub',
+        ext='.png',
+        int2str_len=3,
+    )
+    nose.tools.assert_equal(im_name, 'img_phase_t000_p100_z010_blub.png')
+    im_name = aux_utils.get_sms_im_name(
+        time_idx=0,
+        channel_name=None,
+        slice_idx=None,
+        pos_idx=10,
+        extra_field=None,
+        ext='.jpg',
+        int2str_len=2,
+    )
+    nose.tools.assert_equal(im_name, 'img_t00_p10.jpg')
+
+
```

Comment:
🥇 
Great job adding tests! My only comment is that I would separate these into two tests, which makes it easier to spot exactly where the code breaks if a test isn't passing. But that's optional.

Reply:
done!